index.bs

```diff
@@ -1558,7 +1558,7 @@ run the following steps:
 ## Dereferencing Model for blob URLs ## {#requestResponseModel}
 
 <div algorithm="resolveURL">
-To <dfn export for="blob URL" lt="resolve">resolve a blob URL</dfn> given a |url| (a [=URL=]), run the following steps:
+To <dfn export for="blob URL">resolve a blob URL</dfn> given a |url| (a [=URL=]), run the following steps:
```

Comment:
With `resolve a blob URL` as text, I'd get rid of the `for` part, as that is redundant now.